human-dynamics-estimator/src/HumanDynamicsEstimator.cpp

```diff
@@ -68,53 +176,293 @@ bool HumanDynamicsEstimator::configure(yarp::os::ResourceFinder &rf)
     humanSensors.removeSensor(iDynTree::ACCELEROMETER, base + "_accelerometer");
```

Comment:
I cannot add a line on 167, but, shouldn't the `checkConsistency` called after setting the options?